src/lwc/utilProgressBar/utilProgressBar.js

```diff
@@ -0,0 +1,20 @@
+import {LightningElement, api} from 'lwc';
+
+export default class UtilProgressBar extends LightningElement {
+    @api title;
+
+    @api actualValue;
+    @api expectedValue;
```

Comment:
This is really minor, "expectedValue" seems a little prescriptive for how the progress bar might be used ... "finalValue" or "completedValue" or something? Not sure if is worth focusing on...

Reply:
Yeah, I agree that we could probably come up with something a little more generic for use in other contexts but after "denominator" (which is maybe worse? :) I don't have a lot of great ideas!  Let me know if you have any options and I'm happy to make an update.